src/OpenTelemetry.Api/Internal/SemanticConventions.cs

```diff
@@ -110,5 +110,17 @@ internal static class SemanticConventions
         public const string AttributeExceptionType = "exception.type";
         public const string AttributeExceptionMessage = "exception.message";
         public const string AttributeExceptionStacktrace = "exception.stacktrace";
+
+        // v1.21.0 Http Semantic Conventions
+        public const string AttributeClientSocketPort = "client.socket.port";
+        public const string AttributeHttpRequestMethod = "http.request.method";
+        public const string AttributeHttpResponseStatusCode = "http.response.status_code";
+        public const string AttributeNetworkProtocolVersion = "network.protocol.version";
+        public const string AttributeServerAddress = "server.address";
+        public const string AttributeServerPort = "server.port";
```

Comment:
nit:
It would be good to add comments to show what the old value for each of these were.

```csharp
public const string AttributeServerPort = "server.port"; // Used to be "net.host.port" (AttributeNetHostPort)
```

Reply:
great idea. this would have saved me so much time.

Reply:
done